seurat_examples/seurat_load_example.R

```diff
@@ -0,0 +1,47 @@
+## Install necessary R packages (should only need to be done once) - uncomment to run
+#install.packages("tidyverse")
```

Comment:
Don't need all of `tidyverse`, seems like you load load `dplyr`

Reply:
I debated that internally. Wasn't sure which way to go.

Reply:
Will be in next push